src/app/(proper_react)/redesign/(authenticated)/user/welcome/scanning/page.tsx

```diff
@@ -0,0 +1,123 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest } from "next/server";
+import { getL10n } from "../../../../../../functions/server/l10n";
+import {
+  getOnerepProfileId,
+  getLatestOnerepScan,
+  setOnerepScan,
+  setOnerepScanResults,
+} from "../../../../../../../db/tables/subscribers";
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../../../../api/utils/auth";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Scanning", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserWelcomeScanning() {
+  const session = await getServerSession(authOptions);
+
+  const current = 0;
+  const total = 672;
+  const percentage = ((current / total) * 100).toFixed(1);
```

Comment:
Q: Is it possible that `total` is `0` here and the universe (and/or multiverse) folds in on itself? Or is there a guard somewhere else for a divide by zero error?

Oh, interesting, I expected a RangeError, but looks like this might just return a `NaN` (in about:blank console and Node REPL):

```js
const percentage = ((0 / 0) * 100).toFixed(1);
console.log(percentage); // 'NaN'
```

Which would possibly give us `<h1>NaN%</h1>` below on L120.

Reply:
This is pretty different now, shouldn't have the same problem.